rwiki/rwiki-tool/tool/src/webapp/WEB-INF/vm/macros.vm

```diff
@@ -219,7 +219,7 @@ ${util.formatDateTime($date,$request)}
 					<span id="totalComments" class="textPanelFooter"></span>
 					#if (${renderBean.canEdit} || ${renderBean.canCreate})
 						<span class="commentsToolBar">
-							<a href="#" id="createCommentLink" onclick="ajaxRefPopup(this,'${util.escapeHtml(${renderBean.newCommentURL})}',0); return false;" >${rlb.jsp_addcomment}</a>
+							<a href="#" id="createCommentLink2" onclick="newEditComment(this, '${util.escapeHtml(${renderBean.newCommentURL})}', false); return false;" >${rlb.jsp_addcomment}</a>
```

Comment:
Is there another id with createCommentLink?

I can see only CSS for
  #createCommentLink {
      background: url(/library/image/silk/comment_add.png) center left no-repeat;
      padding:3em 1.6em;
  }

but not for createCommentLink2

Is this OK?

Reply:
I'm glad you spotted this. I think I made a mistake when I pushed the branch. Thanks.